java/src/main/java/org/liberator/ratdriver/control/OperaControl.java

```diff
@@ -1,36 +1,56 @@
 package org.liberator.ratdriver.control;
 
+import lombok.Getter;
+import lombok.Setter;
 import org.liberator.ratdriver.preferences.BasePreferences;
 import org.liberator.ratdriver.preferences.OperaPreferences;
+import org.liberator.ratdriver.settings.BaseSettings;
+import org.liberator.ratdriver.settings.FirefoxSettings;
 import org.liberator.ratdriver.settings.OperaSettings;
 import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.opera.OperaDriver;
 import org.openqa.selenium.opera.OperaDriverService;
 import org.openqa.selenium.opera.OperaOptions;
 
+import java.io.File;
+import java.util.Arrays;
+
+import static org.openqa.selenium.remote.CapabilityType.*;
+
 public class OperaControl extends BrowserControl {
 
     //region Public Properties
 
     /**
      * Holds the preset values for Opera Options
      */
+    @Getter
+    @Setter
     public OperaOptions Options = null;
 
```

Comment:
not sure we need all of the ` = null` on the initialisation as it will automatically set the default

Reply:
Agreed, will remove

Reply:
Done